environment/index.js

```diff
@@ -443,6 +444,24 @@ RainbowConfig.addOptions(
     ],
 );
 
+// Testing command
+const dangerCommand = program.command('DANGER');
```

Comment:
Could we add description to DANGER command and submit_invalid_near_block command?